update kv keys release notes feb (#1032)
Azure/azure-sdk: releases/2020-02/java.md

### KeyVault Keys ([Changelog]())

> - `KeyVaultKey` model can be instantiated using `fromKeyId(String keyId, JsonWebKey jsonWebKey)` and `fromName(String name, JsonWebKey jsonWebKey)` methods on the `KeyVaultKey` model.

Reviewer: ```suggestion
- `KeyVaultKey` model can be instantiated using factory methods `fromKeyId(String keyId, JsonWebKey jsonWebKey)` and `fromName(String name, JsonWebKey jsonWebKey)`.
```

> - Allows `KeyEncryptionKeyClientBuilder` to consume `KeyVaultKey` model and build `KeyEncryptionKey` and `AsyncKeyEncryptionKey` via`buildKeyEncryptionKey(KeyVaultKey key) ` and `buildAsyncKeyEncryptionKey(KeyVaultKey key)` methods respectively.

Reviewer: ```suggestion
- `KeyEncryptionKeyClientBuilder` can consume `KeyVaultKey` and build `KeyEncryptionKey` and `AsyncKeyEncryptionKey` via `buildKeyEncryptionKey(KeyVaultKey key)` and `buildAsyncKeyEncryptionKey(KeyVaultKey key)` methods respectively.
```
